<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Lab_Title">
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>三维扫描架控制程序</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>58</width>
          <height>60</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="GB_ComSetting">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="title">
         <string>串口连接设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="Lab_CurCOM">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>当前端口:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lab_COMName">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_COM">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>打开串口</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Lab_LinkState">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>未连接</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="GB_RealTimeParam">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>实时参数</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="Lab_ControllerNum">
           <property name="text">
            <string>控制器编号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_ControllerNum"/>
         </item>
         <item>
          <widget class="QLabel" name="Lab_LimitSW">
           <property name="text">
            <string>限位开关</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_LimitSW"/>
         </item>
         <item>
          <widget class="QLabel" name="Lab_ScramSW">
           <property name="text">
            <string>急停开关</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_ScramSW"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Lab_MotorPattern">
           <property name="text">
            <string>模式选择</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_MotorPattern">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="C_PowerOnZero">
           <property name="text">
            <string>启用上电回零（开机后自动启动反转回到零点限位停止）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="C_PosControl">
           <property name="text">
            <string>启动位置控制（运行距离控制）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_Model4Num"/>
         </item>
         <item>
          <widget class="QLabel" name="Lab_Model4Num">
           <property name="text">
            <string>按次往返模式：往返次数最大（254）</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="C_SingleSW">
           <property name="text">
            <string>启动单开关（正运行）触发运行功能，不需要负运行开关</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="C_StartPowerOn">
           <property name="text">
            <string>启用开机自动运行（如果和开机回零合用，则先回零）</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="Lab_TravelDistanceNum">
           <property name="text">
            <string>行进距离总脉冲数（最大16777214）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_TravelDistanceNum">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Lab_OperatingSpeedBase">
           <property name="text">
            <string>运行速度基数（最小256最大65535）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_OperatingSpeedBase">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="Lab_SpeedUpNum">
           <property name="text">
            <string>加速脉冲数（最大65535）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_SpeedUpNum">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Lab_SlowDownNum">
           <property name="text">
            <string>减速脉冲数（最大65535）</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="CB_SlowDownNum">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>108</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QComboBox" name="CB_OutputFrequency">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="C_ReadOnly">
           <property name="text">
            <string>启用模式，正运行，负运行开关输入口功能失效，状态只读（可用于对其他输入信号检测）</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>108</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="GB_ComRTControllModel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>串口实时控制模式</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="Btn_Forward">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>正向运动</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_Reverse">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>反向运动</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_Stop">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>停止</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="GB_StateQuery">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>状态查询</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QComboBox" name="CB_State">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>22</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Btn_Query">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="Lab_Send">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>发送的指令</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_Send">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Lab_Receive">
        <property name="text">
         <string>接收的应答</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_Receive">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="QA_ComSettings"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="QA_About"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="QA_ComSettings">
   <property name="text">
    <string>串口设置</string>
   </property>
  </action>
  <action name="QA_About">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
